### PR TITLE
Add IPerfManager documentation

### PR DESCRIPTION
**Pull request by MSNev**
Add documentation for the new IPerfManager, IPerfEvent interfaces added as part of v2.5.7

### CODE REVIEW

**Comment by MSNev** [comment]
I wasn't thinking about this as a support ticket reduction, just documenting the feature.

As such I've not considered where it would appear under the TOC, at this point this will just be available on github in the docs folder. I'll reach out to Rashmi to work out where in the AzureDocs this can live.

File: README.md [hunk 1]
```diff
@@ -274,6 +274,9 @@ Most configuration fields are named such that they can be defaulted to falsey. A
...
 | distributedTracingMode | `DistributedTracingModes.AI` | Sets the distributed tracing mode. If AI_AND_W3C mode or W3C mode is set, W3C trace context headers (traceparent/tracestate) will be generated and included in all outgoing requests. AI_AND_W3C is provided for back-compatibility with any legacy Application Insights instrumented services.
 | enableUnhandledPromiseRejectionTracking | false | If true, unhandled promise rejections will be autocollected and reported as a javascript error. When disableExceptionTracking is true (dont track exceptions) the config value will be ignored and unhandled promise rejections will not be reported.
+| enablePerfMgr | false | [Optional] When enabled this will create local perfEvents for code that has been instrumented to emit perfEvents (via the doPerf() helper) only when this value is true. This can be used to identify performance issues within the SDK based on your usage or optionally within your own instrumented code. [More details are available by the basic documentation](./docs/PerformanceMonitoring.md). Since v2.5.7
```
Comment (kryalama) [inline README.md:277]: Nit picking. "When this value is true" (redundancy)

File: README.md [hunk 2]
```diff
@@ -274,6 +274,9 @@ Most configuration fields are named such that they can be defaulted to falsey. A
...
+| perfEvtsSendAll | false | [Optional] When _enablePerfMgr_ is enabled and the [IPerfManager](https://github.com/microsoft/ApplicationInsights-JS/blob/master/shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IPerfManager.ts) fires a [INotificationManager](https://github.com/microsoft/ApplicationInsights-JS/blob/master/shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/INotificationManager.ts).perfEvent() this flag determines whether the event is actually fired (and sent to all listeners) for all events (true) or only for 'parent' events (false &lt;default&gt;), where parent [IPerfEvent](https://github.com/microsoft/ApplicationInsights-JS/blob/master/shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IPerfEvent.ts)s are determined by whether the event has an identified _parent_ event. Since v2.5.7
```
Comment (kryalama) [inline README.md:278]: Nice to have: Definition/example of 'parent' events in the Performance Monitoring document

File: README.md (hunk 1 above)
Comment (MSNev) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (MSNev) [reply README.md]: Reworded

**Pull request merged**